test/cmdlineTests/yul_string_format_ascii_bytes32_from_number/output.json

```diff
@@ -6,6 +6,7 @@
  *******************************************************/
 
 
+/// @src 0:78,159
 object \"C_11\" {
     code {
         mstore(64, 128)
```

Comment:
We should have the source location of the constructor here.

Reply:
At least for the constructor function itself, for the other lines it is probably better to still be at contract scope.